Makefile

```diff
@@ -11,5 +11,6 @@ theme: mtheme
 
 clean:
 	latexmk -c
+	rm -rf _minted-slides/ *.sty *.snm *.vrb *.nav
```

Comment:
*.sty files did not regenerate after I installed all deps. I had to remove them completely.